diff --git a/feeds/packages/net/sing-box/Makefile b/feeds/packages/net/sing-box/Makefile
--- a/feeds/packages/net/sing-box/Makefile
+++ b/feeds/packages/net/sing-box/Makefile
@@ -48,7 +48,6 @@ endef
 define Package/sing-box-tiny
   $(Package/sing-box-default)
   TITLE+= (tiny)
-  PROVIDES:=sing-box
   VARIANT:=tiny
   CONFLICTS:=sing-box
 endef
 
diff --git a/package/custom/OpenWrt-nikki/mihomo-alpha/Makefile b/package/custom/OpenWrt-nikki/mihomo-alpha/Makefile
--- a/package/custom/OpenWrt-nikki/mihomo-alpha/Makefile
+++ b/package/custom/OpenWrt-nikki/mihomo-alpha/Makefile
@@ -34,7 +34,6 @@ define Package/mihomo-alpha
   TITLE:=A rule based proxy in Go.
   URL:=https://wiki.metacubex.one
   DEPENDS:=$(GO_ARCH_DEPENDS) +ca-bundle +ip-full +kmod-inet-diag +kmod-tun
-  PROVIDES:=mihomo
   CONFLICTS:=mihomo-meta
   VARIANT:=alpha
   ALTERNATIVES:=\
diff --git a/package/custom/OpenWrt-nikki/mihomo-meta/Makefile b/package/custom/OpenWrt-nikki/mihomo-meta/Makefile
--- a/package/custom/OpenWrt-nikki/mihomo-meta/Makefile
+++ b/package/custom/OpenWrt-nikki/mihomo-meta/Makefile
@@ -35,7 +35,6 @@ define Package/mihomo-meta
   URL:=https://wiki.metacubex.one
   DEPENDS:=$(GO_ARCH_DEPENDS) +ca-bundle +ip-full +kmod-inet-diag +kmod-tun
   PROVIDES:=mihomo
-  CONFLICTS:=mihomo-alpha
   VARIANT:=meta
   DEFAULT_VARIANT:=1
   ALTERNATIVES:=\
